sirepo/job_supervisor.py

```diff
@@ -809,3 +854,75 @@ def send(self):
 
     def __hash__(self):
         return hash((self.opId,))
+
+
+def _purge_free_simulations_set():
+    def tomorrow_2am_mst():
```

Comment:
To clarify, the start will be something like next 2am and the period will be 24hours?

I hope none of the physicists see my frequency vs period blunder:)

Reply:
We'll enter a "start" and a period. At startup, we'll run it immediately, because we don't know how long it is down, and then we'll schedule the next one at today + start if > now or today + start + period * N, where N is large enough to be greater than now. For example, if period is 1:0:0 and start is 10:0:0 and the supervisor starts at 18:0:0 UTC, then we would have the next purging event at 19:0:0 UTC or in the above equation N would be 9.